core/grid.js

```diff
@@ -221,6 +234,20 @@ Blockly.Grid.createDom = function(rnd, gridOptions, defs) {
           {'stroke': gridOptions['colour']}, gridPattern);
     }
     // x1, y1, x1, x2 properties will be set later in update.
+  } else if (gridOptions['image'] && gridOptions['image']['path']) {
+    /*
+      <pattern id="blocklyGridPattern837493" patternUnits="userSpaceOnUse"
+              width="IMAGE_WIDTH" height="IMAGE_HEIGHT">
+        <image xlink:href="IMAGE_URL" style="IMAGE_STYLE" />
+      </pattern>
+    */
+    var gridImageOptions = gridOptions['image'];
+    var { path, width, height, ...rest } = gridImageOptions;
```

Comment:
No spread in ES5.

Reply:
gridImageOptions can be null. That would throw when you do ``var { ... } = null``.